### PR TITLE
added twitter link

### LINKED ISSUE #210: [CHORE] add twitter link in README and website
### What would you like to share or ask?

I have created a twitter account for LinksHub so people can tag us and share their experience. 

Add this link [https://twitter.com/the_linkshub](https://twitter.com/the_linkshub) in the README.md file as well in the site with twitter icon (You can also give us a follow ;)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by yashnirmal**
## Fixes Issue
Closes #210

## Changes proposed

Added Twitter link to the website and readme

### CODE REVIEW

**Comment by yashnirmal** [comment]
I created a new fork, and then made changes. After that I deleted the package-lock.json file and push the commit

**Comment by rupali-codes** [comment]
@yashnirmal You have closed and then reopened the same PR ;D.
let's connect on discord, let me know whenever you are available

**Comment by yashnirmal** [comment]
@rupali-codes I have sent you a hi on discord

**Comment by rupali-codes** [comment]
> @rupali-codes I have sent you a hi on discord

yes got it, let's do it after the space

File: README.md [hunk 1]
```diff
@@ -34,6 +34,17 @@ You can see the live demo at: https://linkshub.vercel.app
 ![tailwind css](https://img.shields.io/badge/tailwind_css-7c3aed?style=for-the-badge&logo=tailwindcss&logoColor=white)
 ![typescript](https://img.shields.io/badge/typescript-7c3aed?style=for-the-badge&logo=typescript&logoColor=white)
 
+
+## Socials 📱
+
+<a href="https://twitter.com/the_linkshub" target="blank">
+<img src="https://cdn-icons-png.flaticon.com/32/733/733579.png">
```
Comment (CBID2) [inline README.md:41]: Don’t forget to add an alt tag to the images

File: README.md [hunk 2]
```diff
@@ -34,6 +34,17 @@ You can see the live demo at: https://linkshub.vercel.app
 ![tailwind css](https://img.shields.io/badge/tailwind_css-7c3aed?style=for-the-badge&logo=tailwindcss&logoColor=white)
 ![typescript](https://img.shields.io/badge/typescript-7c3aed?style=for-the-badge&logo=typescript&logoColor=white)
 
+
+## Socials 📱
+
```
Comment (CBID2) [inline README.md:39]: See previous comment.

File: README.md (hunk 1 above)
Comment (CBID2) [inline README.md:41]: Add alt tag here

File: README.md [hunk 3]
```diff
@@ -34,6 +34,17 @@ You can see the live demo at: https://linkshub.vercel.app
...
+<a href="https://twitter.com/the_linkshub" target="blank">
+<img src="https://cdn-icons-png.flaticon.com/32/733/733579.png">
+</a>
+<a href="https://discord.com/invite/NvK67YnJX5" target="blank">
+<img src="https://cdn-icons-png.flaticon.com/32/5968/5968756.png">
```
Comment (CBID2) [inline README.md:44]: Add alt tag

File: README.md (hunk 1 above)
Comment (CBID2) [inline README.md:41]: Add alt tag

**Comment by yashnirmal** [comment]
Hey @CBID2 , I have added the alt property in img tags
Is it good now?

File: README.md [hunk 4]
```diff
@@ -34,6 +34,17 @@ You can see the live demo at: https://linkshub.vercel.app
...
 
+
+## Socials 📱
+
+<a href="https://twitter.com/the_linkshub" target="blank">
+<img src="https://cdn-icons-png.flaticon.com/32/733/733579.png" alt="twitter">
+</a>
+<a href="https://discord.com/invite/NvK67YnJX5" target="blank">
+<img src="https://cdn-icons-png.flaticon.com/32/5968/5968756.png" alt="discord">
+</a>
```
Comment (CBID2) [inline README.md:45]: Looks great @yashnirmal! 😄

**Comment by CBID2** [comment]
> Hey @CBID2 , I have added the alt property in img tags Is it good now?

Yup!

**Comment by rupali-codes** [comment]
thanks Chrissy

**Comment by CBID2** [comment]
> thanks Chrissy

You're welcome! 😄

**Pull request merged**